packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -99,6 +99,7 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     return post?.title;
   };
   const seo: NextSeoProps = {
+    canonical: post?.slug,
```

Comment:
It's not enough for the canonical isn't it? You miss the `/posts/` part

Reply:
You're absolutely right 😶‍🌫️